templates/events/events_thanks.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base.html" %}
+{% load boxes %}
```

Comment:
Do we need this line? I assume we can remove it since we didn't define a `{% block right_sidebar %}` block here.